Packs/JoeSecurity/Integrations/JoeSecurityV2/JoeSecurityV2.py

```diff
@@ -0,0 +1,527 @@
+from typing import Generator, Tuple
+
+import jbxapi
+
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+
+
+class Client(jbxapi.JoeSandbox):
+    def __init__(self, apikey: str = '', base_url: str = '', accept_tac: bool = True,
+                 verify_ssl: bool = True,
+                 proxy: bool = False, create_relationships: bool = False,
+                 reliability: str = DBotScoreReliability.C):
+        self.reliability = reliability
+        self.create_relationships = create_relationships
+        super().__init__(apikey=apikey, apiurl=base_url, accept_tac=accept_tac, verify_ssl=verify_ssl, proxies=proxy)
+
+    def analysis_info_list(self, web_ids: List[str]) -> List[Dict[str, Any]]:
+        """
+             A wrapper function supporting a list of webids to query.
+
+             Args:
+                web_ids: List(str): List of analysis webids to query.
+
+             Returns:
+                 result: List(Dict(str, any)): List of analysis info result.
+        """
+        return [self.analysis_info(webid=web_id) for web_id in web_ids]
+
+
+''' HELPER FUNCTIONS '''
+
+
+def pagination(args: Dict[str, Any], results: Generator) -> List:
```

Comment:
```suggestion
def paginate(args: Dict[str, Any], results: Generator) -> List:
```
nice, please open an issue for adding a similar implementation in CSP